docs/federation.md

```diff
@@ -133,22 +138,41 @@ This query comes with the following variables:
       "__typename": "Show",
       "id": 5
     }
-  ]        
-} 
+  ]
+}
 ```
 
-The Reviews DGS needs to implement an `entity fetcher` to handle this query.
-An entity fetcher is responsible for creating an instance of a `Show` based on the representation in the `_entities` query above.
-The DGS framework does most of the heavy lifting, and all we have to do is provide the following:
+The Reviews DGS needs to implement an entity fetcher to handle this query.
+
+- **Entity fetcher**: A method annotated with [`@DgsEntityFetcher`](https://javadoc.io/doc/com.netflix.graphql.dgs/graphql-dgs/latest/com/netflix/graphql/dgs/DgsEntityFetcher.html) that takes a key and returns a single instance of the entity or null.
```

Comment:
what are the implications of returning the entity or null? which one is preferred?

Reply:
I think it also makes sense to mention that the entity resolver must return some value for every key.